.changeset/shiny-tomatoes-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+"@openfun/cunningham-react": minor
+---
+
+homogenize form fields colors
```

Comment:
I don't realy understand, the commit talk about using the same greyscale but it used 200, 300, 600, 800 and 900

Reply:
across the inputs. If you look at form examples before and after this commits you will see that before the grey colors were not homogeneous across inputs at all